tox.ini

```diff
@@ -7,7 +7,7 @@ deps =
     -r{toxinidir}/requirements/test.txt
 commands =
     coverage run setup.py test
-    coverage report
+    python -m coverage xml
```

Comment:
why was this change needed?

Reply:
Version 2 of Codecov Action does not generate the xml file which version 1 previously did for us hence this change......
Reference PR by Ned: [PR](https://github.com/edx/XBlock/pull/560)